setup.cfg

```diff
@@ -1,4 +1,87 @@
-# This file should be used as the pep8/pycodestyle and autopep8 config
+###############################################################################
+# Setuptools
+###############################################################################
+
+[metadata]
+name = DIRAC
+description = DIRAC is an interware, meaning a software framework for distributed computing.
+long_description = file: README.rst
+license = GPL-3.0-only
+classifiers =
+    License :: OSI Approved :: GNU General Public License v3 (GPLv3)
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3.8
+    Programming Language :: Python :: 3.9
+
+[options]
+python_requires = >=3.8
+package_dir=
+    =src
+packages = find:
+install_requires =
+    certifi
+    diraccfg
+    future
+    gfal2-python
+    M2Crypto
+    pexpect
+    psutil
+    pyasn1
+    pyasn1-modules
+    python-dateutil
+    requests
+    setuptools
```

Comment:
It's used at runtime for the `pkg_resources` module that gives the version metadata in `DIRAC/__init__.py` (for historical reasons it is `setuptools` package instead of being standalone).

Reply:
Hum, I guess we are going to distribute `wheels`, so whatever is required for the build wont be shipped, correct ?

Reply:
Yes. In practice almost everyone will have `pip` and therefore `setuptools` already installed but it's "correct" to include it here to make sure.